src/index.html

```diff
@@ -19,9 +19,423 @@
       rel="stylesheet"
       href="styles/index.scss"
     />
+    <link
+      rel="stylesheet"
+      href="styles/main.scss"
+    />
   </head>
-
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__container">
+        <div class="header__logo">
+          <a
+            href="#moyo"
+            class="header__logo-link"
+          >
+            <img
+              src="images/logo.png"
+              alt="moyo logo"
+              class="header__logo-img"
+            />
+          </a>
+        </div>
+        <nav class="header__nav nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a
+                href="#apple"
+                class="nav__link is-active"
+              >
+                Apple
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#samsung"
+                class="nav__link"
+              >
+                Samsung
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#smartphones"
+                class="nav__link"
+              >
+                Smartphones
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                data-qa="nav-hover"
+                href="#laptops"
+                class="nav__link"
+              >
+                Laptops & Computers
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#gadgets"
+                class="nav__link"
+              >
+                Gadgets
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#tablets"
+                class="nav__link"
+              >
+                Tablets
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#photo"
+                class="nav__link"
+              >
+                Photo
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#video"
+                class="nav__link"
+              >
+                Video
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main class="products">
+      <div class="products__wrapper">
+        <div
+          class="products__card card"
+          data-qa="card"
+        >
+          <div class="card__container">
```

Comment:
You are repeating a lot of code for each product. Consider using a templating engine or a component-based library or framework to avoid repetition. This will make your code more maintainable.